.github/workflows/static.yml

```diff
@@ -15,11 +15,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Remove phpspec
-        run: composer remove --dev friends-of-phpspec/phpspec-code-coverage phpspec/phpspec
-
       - name: PHPStan
-        uses: OskarStark/phpstan-ga@0.12.32
+        uses: docker://oskarstark/phpstan-ga
```

Comment:
can you please rebase on 2.x so we only see the changes about php 8?

Reply:
Yes. I did just now.

Reply:
thanks, neat!